packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -300,12 +300,12 @@ storiesOf('Picker', module)
     'long item text',
     () => (
       <Picker label="Test" onSelectionChange={action('selectionChange')}>
-        <Item key="short">One</Item>
-        <Item key="long">your text here long long long long</Item>
-        <Item key="underscores">your_text_here_long_long_long_long</Item>
-        <Item key="hyphens">your-text-here-long-long-long-long</Item>
-        <Item key="singleWord">supercalifragilisticexpialidocious</Item>
-        <Item key="always">This item is very long and word wraps poorly</Item>
+        <Item key="One">One</Item>
+        <Item key="your text here long long long long">your text here long long long long</Item>
+        <Item key="underyour_text_here_long_long_long_longscores">your_text_here_long_long_long_long</Item>
```

Comment:
was this key being `underyour_text_here_long_long_long_longscores` rather than `your_text_here_long_long_long_long` intentional? I think this is fine since it is still related enough, but just curious

Reply:
sorry it's a typo, will fix it shortly